src/incubator/WheelPicker/Item.tsx

```diff
@@ -4,11 +4,11 @@ import Animated, {interpolateColor, useAnimatedStyle} from 'react-native-reanima
 import Text, {TextProps} from '../../components/text';
 import TouchableOpacity from '../../components/touchableOpacity';
 import {Colors, Spacings} from '../../../src/style';
-import {asBaseComponent} from '../../commons/new';
+import {asBaseComponent, asClassComponent} from '../../commons/new';
 import {WheelPickerAlign} from './types';
 
 const AnimatedTouchableOpacity = Animated.createAnimatedComponent(TouchableOpacity);
-const AnimatedText = Animated.createAnimatedComponent(Text);
+const AnimatedText = Animated.createAnimatedComponent(asClassComponent(Text));
```

Comment:
Can't we just use our Text component with `animated = true` instead?

Reply:
Our Text supports `animated` prop to render React Native Animated
But in this case we use Reanimated Text. 
I want to add a `reanimated` prop to Text that will return a Reanimated Text component. 
But i didn't want to do it in this PR. 

in any case, `asClassComponent` is a good HOC, cause it's not the first time I came across this issue
(we had it after migrating AgendaList to a function component)